### PR TITLE
ViPNAS ResNet

### PR DESCRIPTION
**Pull request by luminxu**
add ViPNAS ResNet models.

https://github.com/luminxu/ViPNAS

### CODE REVIEW

**Review by jin-s13 — APPROVED** [review]

**Comment by jin-s13** [comment]
README will be updated after code merge.

File: configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/coco/vipnas_coco.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!-- [BACKBONE] -->
+
+<details>
+<summary align="right">ViPNAS (CVPR'2021)</summary>
+
+```bibtex
+@article{xu2021vipnas,
```
Comment (innerlee) [inline configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/coco/vipnas_coco.md:7]: For new papers, an item should be added here https://github.com/open-mmlab/mmpose/tree/master/docs/papers

File: mmpose/models/heads/vipnas_heatmap_simple_head.py [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+        Args:
+            output (torch.Tensor[NxKxHxW]): Output heatmaps.
+            target (torch.Tensor[NxKxHxW]): Target heatmaps.
+            target_weight (torch.Tensor[NxKx1]):
+                Weights across different joint types.
+        """
+
+        accuracy = dict()
+
+        if self.target_type == 'GaussianHeatMap':
```
Comment (innerlee) [inline mmpose/models/heads/vipnas_heatmap_simple_head.py:186]: Heatmap is a word, but not "Heat Map"

**Review by innerlee — APPROVED** [review]

File: mmpose/models/backbones/vipnas_resnet.py [hunk 3]
```diff
@@ -0,0 +1,597 @@
...
+            out += identity
+
+            return out
+
+        if self.with_cp and x.requires_grad:
+            out = cp.checkpoint(_inner_forward, x)
+        else:
+            out = _inner_forward(x)
+
+        out = self.relu(out)
```
Comment (ly015) [inline mmpose/models/backbones/vipnas_resnet.py:169]: Why this line is outside the _inner_forward?

Comment (jin-s13) [reply mmpose/models/backbones/vipnas_resnet.py:169]: That's copied from backbones/resnet.py

Comment (ly015) [reply mmpose/models/backbones/vipnas_resnet.py:169]: I see. Do you know the reason?

Comment (jin-s13) [reply mmpose/models/backbones/vipnas_resnet.py:169]: Maybe it is because `relu` does not have any params.
